homeassistant/components/device_tracker/googlehome.py

```diff
@@ -23,6 +23,8 @@
 async def async_setup_scanner(hass, config, async_see, discovery_info=None):
     """Validate the configuration and return a Google Home scanner."""
     if discovery_info is None:
+        _LOGGER.error(
```

Comment:
I would make it a warning. This is just a config mistake and not intended use.

Reply:
Changed :)